tests/test_cmdline.py

```diff
@@ -22,6 +30,19 @@ def test_commandline_explicit_interp(tmpdir):
         str(tmpdir.join('venv'))
     ])
 
+# On Windows, path with Chinese characters are not working without a Chinese code page
+@pytest.mark.skipif("get_code_page() not in (None, 936, 950)")
+def test_commandline_non_ascii_path(tmpdir):
+    target_path = str(tmpdir.join('venv中文'))
+    if sys.version_info[0] == 2:
```

Comment:
Why not use a non-ASCII character that's present in the default code page on Appveyor, rather than messing about changing the locale of the system? Something like an accented character (é) should be available in the default codepage.